_includes/live-demos/rtc/index.js

```diff
@@ -0,0 +1,320 @@
+];
+
+/*
+ * A set of pregenerated fake users. The user details are passed to TinyMCE
+ * using the getUserDetails endpoint/server so full names or other details can
+ * be presented in the UI.
+ */
+const fakeUsers = [
+  {_id: 'melyna_hartmann28', firstname: 'Melyna', lastname: 'Hartmann', fullname: 'Miss Melyna Hartmann', email: 'melyna62@hotmail.com'},
+  {_id: 'nettie_kuhic', firstname: 'Nettie', lastname: 'Kuhic', fullname: 'Nettie Kuhic Jr.', email: 'nettie.kuhic@gmail.com'},
+  {_id: 'piper6', firstname: 'Piper', lastname: 'Dare', fullname: 'Piper Dare', email: 'piper_dare@hotmail.com'},
+  {_id: 'ansley84', firstname: 'Ansley', lastname: 'Schaden', fullname: 'Ansley Schaden', email: 'ansley_schaden89@gmail.com'},
+  {_id: 'brandy_reinger24', firstname: 'Brandy', lastname: 'Reinger', fullname: 'Brandy Reinger', email: 'brandy90@hotmail.com'},
+  {_id: 'selmer.luettgen72', firstname: 'Selmer', lastname: 'Luettgen', fullname: 'Selmer Luettgen', email: 'selmer.luettgen69@yahoo.com'},
+  {_id: 'mikayla_dibbert', firstname: 'Mikayla', lastname: 'Dibbert', fullname: 'Mikayla Dibbert', email: 'mikayla.dibbert@yahoo.com'},
+  {_id: 'mabelle.robel39', firstname: 'Mabelle', lastname: 'Robel', fullname: 'Mabelle Robel', email: 'mabelle73@gmail.com'},
+  {_id: 'dino.davis', firstname: 'Dino', lastname: 'Davis', fullname: 'Dino Davis', email: 'dino89@hotmail.com'},
+  {_id: 'josie.bergstrom', firstname: 'Josie', lastname: 'Bergstrom', fullname: 'Josie Bergstrom Jr.', email: 'josie4@yahoo.com'},
+  {_id: 'guadalupe48', firstname: 'Guadalupe', lastname: 'Reilly', fullname: 'Guadalupe Reilly', email: 'guadalupe51@gmail.com'},
+  {_id: 'jayde92', firstname: 'Jayde', lastname: 'Gutkowski', fullname: 'Jayde Gutkowski', email: 'jayde11@gmail.com'}
+];
+
+/* Generate a document ID */
+const documentID = randomString() + Date.now();
+
+/* Generating random encryption key and keyHint for demonstration purposes */
+const demoEncryptionKeyPair = {
+  key: randomString(),
+  keyHint: randomString(),
+};
+
+/* Retrieve random two users, one for each editor. */
+const sessionUsers = getTwoRandomInt(fakeUsers.length);
+const currentUser1 = fakeUsers[sessionUsers[0]]._id,
+      currentUser2 = fakeUsers[sessionUsers[1]]._id;
+
+/*
+  * Add the "current user" of the editor above each instance, outside the
+  * editor, for reference purposes only
+  */
+document.getElementById('fakedemouser1').innerText = fakeUsers.find(
+  (user) => {
+    return user._id === currentUser1;
+  }
```

Comment:
```suggestion
  (user) =>  user._id === currentUser1
```
This can be simplified to this if we're using arrow functions.